src/main/mule/aggregators/aggregator-sdlc-metrics.xml

```diff
@@ -24,7 +24,10 @@ http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/
 				<flow-ref doc:name="JIRA Metrics Flow Reference" doc:id="d42691e3-59fd-475b-a4ef-25b62a046c34" name="collector-jira-metrics-stories-flow" />
 			</route>
 			<route>
-				<flow-ref doc:name="Splunk Metrics Flow Reference" doc:id="d27ee5f5-d6a3-4d16-bc9c-c3d22befbdc9" name="collector-splunk-metrics-flow" />
+				<flow-ref doc:name="Splunk Metrics Flow Reference" doc:id="d27ee5f5-d6a3-4d16-bc9c-c3d22befbdc9" name="collector-azuredevops-metrics-flow" />

```

Comment:
This seems to be a mistake, the Splunk Metrics Flow should point to the Splunk collector not to the new Azure collector

Reply:
Yep, thanks for the catch.   Must have been a copy/paste mistake

Reply:
correction made to branch.   Please review again.